tests/test_uncrustify/test.py

```diff
@@ -56,6 +56,7 @@ def build(self, test_input, test_lang, test_config, test_expected):
         self.test_config = test_config
         self.test_expected = test_expected
         self.test_xfail = False
+        self.multiple_configs = False
```

Comment:
Your argument naming is inconsistent. That said, I wonder if this even belongs here; the use case for passing multiple configurations to a formatting unit test seems very minimal. A front-end test might be a better choice.

Failing that, I would strongly encourage to instead specify the new functionality as `@list-of-configs`. In fact, I would very strongly consider pushing that processing into uncrustify itself rather than add a new `-C` option, as response files (denoted with `@`) are a widespread convention.